ReSharper.FSharp/test/src/FSharp.Tests/QuickFixes/ReplaceExpressionWithOperatorTest.fs

```diff
@@ -0,0 +1,39 @@
+﻿namespace JetBrains.ReSharper.Plugins.FSharp.Tests.Features
+
+open JetBrains.ReSharper.FeaturesTestFramework.Intentions
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+open JetBrains.ReSharper.Plugins.FSharp.Tests
+open NUnit.Framework
+
+type ReplaceExpressionWithOperatorTest() =
+    inherit FSharpQuickFixTestBase<ReplaceExpressionWithOperatorFix>()
+
+    override x.RelativeTestDataPath = "features/quickFixes/replaceExpressionWithOperator"
+
+    [<Test>] member x.``Id - Lambda 1``() = x.DoNamedTest()
+    [<Test>] member x.``Id - Lambda 2 - add whitespaces``() = x.DoNamedTest()
+    [<Test>] member x.``Id - Lambda body 1``() = x.DoNamedTest()
+    [<Test>] member x.``Id - Lambda body 2 - add whitespace``() = x.DoNamedTest()
+
+    [<Test>] member x.``Fst - Lambda 1``() = x.DoNamedTest()
+    [<Test>] member x.``Fst - Lambda 2 - tuple``() = x.DoNamedTest()
+    
+    [<Test>] member x.``Snd - Lambda 1``() = x.DoNamedTest()
+    [<Test>] member x.``Snd - Lambda 2 - tuple``() = x.DoNamedTest()
```

Comment:
What does these tests add over `Fst` ones? They seem to be testing exactly the same logic in the same way.